.github/actions/build-image/action.yml

```diff
@@ -92,3 +92,4 @@ runs:
       with:
         docker-img: ${{ inputs.docker-img }}
         publish-version: ${{ inputs.branch-name }}
+        url-docker-registry: ${{ env.URL_DOCKER_REGISTRY }}
```

Comment:
This seems like the right way to go. 
But note that now we will again have a surprising fail scenario if this environment variable is not set. 

Let's repeat the earlier process and add *url-docker-registry* as explicit input of this action, and then pass it down the chain.

Reply:
Done! Thanks a lot for the feedback! 
I still kept the same pattern, optional with default value. Let me know if that works!